srcpkgs/dinit/template

```diff
@@ -1,22 +1,22 @@
 # Template file for 'dinit'
 pkgname=dinit
-version=0.19.4
+version=0.20.0
 revision=1
-build_style=meson
-configure_args="
- -Ddinit-sbindir=/usr/bin
- -Dshutdown-prefix=dinit-
- -Dunit-tests=true
- -Digr-tests=true
- -Db_ndebug=false"
+build_style=gnu-configure
```

Comment:
The `configure` script provided by dinit is not produced by autotools, but that is the intended target of the `gnu-configure' build style. Use the `configure` build style instead, or the `gnu-makefile` style with a custom `do_configure` if that is more appropriate.